exe/PowershellShim-Helper.ps1

```diff
@@ -6,7 +6,12 @@
 #
 #  > $Json | ConvertFrom-json -Type PSObject
 
-Add-Type -AssemblyName System.ServiceModel.Web, System.Runtime.Serialization
+try {
+  Add-Type -AssemblyName System.ServiceModel.Web, System.Runtime.Serialization
+} catch {
+  throw "Could not load System.ServiceModel.Web or System.Runtime.Serialization, needed for JSON parsing. Please install .NET Framework 3.5 or greater."
```

Comment:
@MFredette I'd be interested in suggestions on what kind of detail this error should go into. Maybe only the 2nd sentence is needed.

Reply:
An alternative proposal: `Could not load assemblies needed for JSON parsing. Please install .NET Framework 3.5 or greater, or rewrite task to use a different input method.`

Reply:
@MikaelSmith - my suggestions are mainly grammatical. If you think folks will be confused about which assemblies, then I'd use the first version, with slight edits: "Could not load System.ServiceModel.Web or System.Runtime.Serialization, which are needed for JSON parsing. Please install .NET Framework 3.5 or greater."

If you think the assemblies are obvious enough, then the second option is good, with some small tweaks: "Could not load the assemblies needed for JSON parsing. Please install .NET Framework 3.5 or greater, or rewrite the task to use a different input method."

Also, it might help to have a noun in there: "PXP agent could not load..." (or whatever the appropriate noun would be. Adding the noun could especially help if this is ever translated, or for people for whom English is not their primary language.

Reply:
Thanks for the review. Updated.